webapp/src/hooks/useLoadFilters.ts

```diff
@@ -0,0 +1,53 @@
+import {useCallback, useEffect} from 'react';
+
+import {filterLabelValuePairAll} from 'pluginConstants/common';
+import {formLabelValuePairs} from 'utils';
+
+import pluginConstants from 'pluginConstants';
+
+import usePluginApi from './usePluginApi';
+
+type UseLoadFiltersParams = {
```

Comment:
```suggestion
type UseLoadFiltersProps = {
```

Reply:
this is a react hook, we name props only for react components